src/pages/program.jsx

```diff
@@ -657,7 +657,7 @@ Technical requirements: Users should feel comfortable using a terminal & Docker
             {isExtravaganza && (
               <ProgramItem
                 dayStr="2022-09-16"
-                startTime="23:59"
+                startTime="23:00"
```

Comment:
Michael could start at 11pm just like Saturday as Creator's Lab is not used after 11.